draft-ietf-taps-interface.md

```diff
@@ -138,16 +138,16 @@ network paths dynamically at runtime. This API enables faster deployment
 of new protocols and protocol features without requiring changes to the
 applications. The specified API follows the Transport Services Architecture
 by providing asynchronous, atomic transmission of messages. It is intended to replace the
-traditional BSD sockets API as the common interface to the
+BSD sockets API as the common interface to the
 transport layer, in an environment where endpoints could select from 
 multiple interfaces and potential transport protocols.
 
 --- middle
 
 # Introduction
 
-This document specifies a modern abstract application programming interface (API) atop the
-high-level architecture for transport services defined in
+This document specifies an abstract application programming interface (API) that follows
+the high-level architecture for transport services defined in
 {{I-D.ietf-taps-arch}}. The Transport Services Architecture supports
```

Comment:
This text I think might be a part of our problem about the difference between ARCH and API in #889. I think we would be served better to be more informative about the way that the API is a key component of the ARCH. ... So, I suggest:
/that follows high-level architecture/that specifies the interface component of  the high-level architecture/.